css/screens/signupPage.css

```diff
@@ -0,0 +1,33 @@
+/* css for signUp page */
```

Comment:
Remove comment.

Reply:
Actually many suggested me to add comments before writing css for better understanding.